survey/dashboards.py

```diff
@@ -0,0 +1,1708 @@
+                    'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.1)',
+                    'marginBottom': '20px'
+                }),
+
+                # Graph Card
+                dbc.Card([
+                    dbc.CardBody([
+                        dcc.Graph(
+                            id='section-breakdown-e',
+                            config={'displayModeBar': False}
+                        )
+                    ])
+                ], style={
+                    'borderRadius': '10px',
+                    'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.1)'
+                })
+            ], width=12)
+        ], style={'marginTop': '20px', 'marginBottom': '20px'}
+        ),
+
+    dbc.CardBody([
+            dash_table.DataTable(
+                id='section-e-stats',
+                columns=[
+                    {'name': 'Statistic', 'id': 'statistic'},
+                    {'name': 'Value', 'id': 'value'}
+                ],
+                style_table={
+                    'width': '100%',
+                    'borderRadius': '1rem',
+                    'overflow': 'hidden',
+                    'border': '1px solid #ddd',
+                    'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.1)',
+                },
+                style_cell={
+                    'textAlign': 'center',
+                    'padding': '10px',
+                    'fontFamily': THEME['font_family'],
+                    'fontSize': '15px',
+                    'border': '1px solid #ddd',
+                    'minWidth': '0px',
+                    'maxWidth': 'none',
+                    'whiteSpace': 'normal',
+                    'height': 'auto'
+                },
+                style_header={
+                    'backgroundColor': 'white',
+                    'fontWeight': 'bold',
+                    'border': '1px solid #ddd',
+                    'whiteSpace': 'normal',
+                    'height': 'auto',
+                    'fontSize': '16px'
+                }
+            )
+        ]),
+
+        dbc.Row([
+            dbc.Col([
+                dbc.Card([
+                    dbc.CardHeader([
+                        html.H5("Pay Band Proportion",
+                                style={
+                                    'fontFamily': THEME['font_family'],
+                                    'color': THEME['text_color'],
+                                    'marginBottom': '0px'
+                                })
+                    ]),
+                    dbc.CardBody([
+                        dcc.Graph(
+                            id='band-chart',
+                            config={'displayModeBar': False}
+                        )
+                    ])
+                ], style={
+                    'borderRadius': '10px',
+                    'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.1)'
+                })
+            ], width=12)
+        ], style={'marginTop': '20px', 'marginBottom': '20px'}),
+
+        dbc.Row([
```

Comment:
A function can be used to generate the configuration of each of the demographic section